DigitalLearningSolutions.Data.Tests/DataServices/CourseDataServiceTests.cs

```diff
@@ -245,5 +245,47 @@ public void GetCourseStatisticsAtCentreForCategoryID_should_return_course_statis
             result.Should().HaveCount(260);
             result.First().Should().BeEquivalentTo(expectedFirstCourse);
         }
+
+        [Test]
+        public void GetDelegateCoursesInfo_should_return_delegate_course_info_correctly()
+        {
+            // When
+            var results = courseDataService.GetDelegateCoursesInfo(20).ToList();
+
+            // Then
+            var enrollmentDate = new DateTime(2019, 04, 11, 14, 33, 37).AddMilliseconds(140);
+            var expected = new DelegateCourseInfo(

```

Comment:
Do we want a helper for this?

Reply:
For creating the object? I didn't think so as it's only used once and none of the other tests in the file use helpers.

Reply:
OK fine, I'll bring it up again if we start initialising multiple of these in the future.